### ISSUE TITLE
UTF-8 Encoding

### ISSUE THREAD

**Issue by karlnicholas** [SFMOCI/openlaw#7]
In the .txt file I'm getting:

SEC. 8.08.  ENFORCEMENT OF ARTICLE  RULES AND REGULATIONS.

But in the website I'm getting:

 § 8.08 ENFORCEMENT OF ARTICLE – RULES AND REGULATIONS.

Notice the hyphen.

**Comment by jasonlally** [comment]
Yes, sorry about this.  Our vendor does not have a way to export from their database in UTF-8, we have to run a script separately to convert before committing to git and that's missing from our last rounds of updates.  Working on fixing that so that each update is run through UTF-8 conversion. Not a perfect solution, but best we can do right now.

**Comment by karlnicholas** [comment]
Cool, sounds good.
